src/Sylius/Behat/Context/Api/Shop/LoginContext.php

```diff
@@ -270,9 +271,9 @@ public function iShouldNotBeAbleToChangeMyPasswordAgainWithTheSameToken(): void
         $this->client->executeCustomRequest($this->request);
 
         // token is removed when used
-        Assert::same($this->client->getLastResponse()->getStatusCode(), 500);
+        Assert::same($this->client->getLastResponse()->getStatusCode(), Response::HTTP_NOT_FOUND);
```

Comment:
I wonder if we should rely (in tests) on fixed values instead of relying on some class constants
```suggestion
        Assert::same($this->client->getLastResponse()->getStatusCode(), 404);
```

Reply:
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad
Class constants > fixed values.

Reply:
Therefore it should depend on the constant inside the context class

Reply:
The possibility of this constant disappearing is low, and in such case we're able to easy replace it.